packages/datagateway-common/src/homePage/homePage.component.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import {
+  withStyles,
+  Theme,
+  WithStyles,
+  Grid,
+  createStyles,
+} from '@material-ui/core';
+import { StyleRules } from '@material-ui/core/styles';
+import Image from '../images/Jasmin4 _DSC7054.jpg';
+import DatagatewayLogoWithText from '../images/datagateway-logo-white.svg';
+import ExploreImage from '../images/explore.jpg';
+import DiscoverImage from '../images/discover.jpg';
+import DownloadImage from '../images/download.jpg';
+// import { UKRITheme } from './theming';
```

Comment:
Will go back to. This wasn't working properly last I tried.

Reply:
Deciding to remove the theming file entirely and instead hard-code the only line that was being imported from it (the bright orange colour)